tests/docker-scripts/Dockerfile

```diff
@@ -19,28 +26,20 @@ RUN fc-list : family;
 COPY .nvmrc .
 
 # Set the default shell so the correct node/npm is used after install
-# https://stackoverflow.com/a/60137919
+# Based on https://stackoverflow.com/a/60137919
```

Comment:
I think I found away to avoid the interactive shell here:

```dockerfile
SHELL ["/bin/bash", "--login", "-c"]
# Nvm appends its config to `.bashrc` if it exists, but the default `.bashrc` 
# bails if not in an interactive shell, which causes `nvm` to never actually get
# configured. To avoid having to use `-i` interactive shell, we overwrite the
# `.bashrc` file with a version that doesn't bail.
# https://github.com/nvm-sh/nvm/issues/1985#issuecomment-813189002
RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
RUN echo 'export NVM_DIR="$HOME/.nvm"' > /root/.bashrc
RUN echo 'source "$NVM_DIR/nvm.sh" --install' >> /root/.bashrc
RUN source /root/.bashrc
RUN nvm install
```